src/Microsoft.Framework.Runtime/NuGet/Utility/VersionUtility.cs

```diff
@@ -251,7 +246,7 @@ public static FrameworkName ParseFrameworkName(string frameworkName)
                     return UnsupportedFrameworkName;
                 }
 
-                version = _emptyVersion;
+                version = identifierPart.Equals(NetPlatformFrameworkIdentifier) ? new Version(5, 0) : _emptyVersion;
```

Comment:
@emgarten came through with a clean way to do this in the new nuget code, so we can keep it! Ignore my previous comment :).